Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -189,6 +200,33 @@ extension CommentRepliesViewController: CommentComposerViewDelegate {
   }
 }
 
+// MARK: - CommentCellDelegate
+
+extension CommentRepliesViewController: CommentCellDelegate {
+  func commentCellDidTapHeader(author _: Comment.Author) {
+    guard AppEnvironment.current.currentUser != nil, featureBlockUsersEnabled() else { return }
+
+    if #available(iOS 15.0, *) {
```

Comment:
Is the plan to only make blocking available for iOS 15+? If I remember correctly, we were planning to drop iOS 14 anyways because iOS 17 is out now, so I think we should just drop iOS 14 entirely instead of adding these if available tags.

Reply:
That's right. I think we just haven't made the effort to prioritize that work with all of the release blockers. That could be something we can tackle separately while we wait on the backend though.

Reply:
I have the bandwidth to work on dropping 14 this week, if that'd be helpful.